Source/player.cpp

```diff
@@ -1062,12 +1060,11 @@ void NextPlrLevel(int pnum)
 		drawhpflag = true;
 	}
 
+	int mana = 128;

```

Comment:
`a lot more unreadable` -> `a lot less readable`

Reply:
The idea here is to have variables initialized with a sensible default to avoid bugs related to unitized variables. I'm open to alternatives :ear:

Reply:
I'm aware of the "variable is not initialized" issue, but it was initialized in the else in this case, so there was no problem.

The best alternative (IMHO) is to extract a function or local lambda that returns only this value.

Reply:
At least for me it makes it much easier to see that this one is not affected by the issue, instead of having to manually evaluate the following code to determine if it's the case or not. Making it easier to weed out any remaining ones once done with the general clean up.

I agree with extracting a method but don't want to expand the work done in this PR to much.